frontend/src/components/popup/confirmation/ConfirmationPopUpController.jsx

```diff
@@ -0,0 +1,27 @@
+import { useState } from 'react'
```

Comment:
I've made mention of this in the Discord, but I'm currently not 100% sure how we would use the controller as most popups are just layered within the jsx, but I've just left it here - should be relatively straightforward in any respect, and would be faster to delete than rewrite.